storage/store_test.go

```diff
@@ -465,8 +465,6 @@ func TestStoreExecuteNoop(t *testing.T) {
 	}
 }
 
-// TestStoreErrorWithIndex verifies that an *errWithIndex returned from the
-// replica is correctly translated into a *proto.Error with a corresponding
 // Index. See the companion test TestBatchWithError.
```

Comment:
Removed the test. `TestBatchErrorWithIndex` tests the same thing closer to
the source.

On Wed, Sep 23, 2015 at 10:53 AM, Tamir Duberstein <notifications@github.com
> wrote:

> In storage/store_test.go
> <https://github.com/cockroachdb/cockroach/pull/2629#discussion_r40212652>:
>
> > @@ -465,8 +465,6 @@ func TestStoreExecuteNoop(t *testing.T) {
> >  	}
> >  }
> >
> > -// TestStoreErrorWithIndex verifies that an *errWithIndex returned from the
> > -// replica is correctly translated into a *proto.Error with a corresponding
> >  // Index. See the companion test TestBatchWithError.
>
> this comment is broken now
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cockroachdb/cockroach/pull/2629/files#r40212652>.
>